app/src/test/java/net/aiscope/gdd_app/repository/SampleRepositorySharedPreferenceTest.kt

```diff
@@ -143,18 +145,47 @@ class SampleRepositorySharedPreferenceTest {
         val yesterday = Calendar.getInstance()
         yesterday.add(Calendar.DAY_OF_YEAR, -1)
 
-        val todaySample = Sample(todayId, HOSPITAL_ID, MICROSCOPIST, status = SampleStatus.ReadyToUpload, createdOn = today)
+        val todaySample = Sample(todayId, HOSPITAL_ID, MICROSCOPIST, DISEASE, status = SampleStatus.ReadyToUpload, createdOn = today)
         val todaySampleJson = """{"id":"1112","healthFacility":"H_St_Pau","status":"ReadyToUpload","createdOn":"$today"}"""
-        val yesterdaySample = Sample(yesterdayId, HOSPITAL_ID, MICROSCOPIST, status = SampleStatus.ReadyToUpload, createdOn = yesterday)
+        val yesterdaySample = Sample(yesterdayId, HOSPITAL_ID, MICROSCOPIST, DISEASE, status = SampleStatus.ReadyToUpload, createdOn = yesterday)
         val yesterdaySampleJson = """{"id":"1113","healthFacility":"H_St_Pau","status":"ReadyToUpload","createdOn":"$yesterday"}"""
 
         whenever(gson.fromJson<SampleDto>(eq(todaySampleJson), any<Type>())).thenReturn(todaySample.toDto())
         whenever(gson.fromJson<SampleDto>(eq(yesterdaySampleJson), any<Type>())).thenReturn(yesterdaySample.toDto())
 
         whenever(store.all()).thenReturn(listOf(sampleOnlyRequiredJson, todaySampleJson, yesterdaySampleJson))
 
-        val sample = subject.last()
+        val sample = subject.lastSaved()
```

Comment:
Should this test be renamed to better reflect what is tested?

Reply:
'stored' vs 'unfinished?'